web/src/admin.ts

```diff
@@ -280,11 +281,18 @@ export function build_page(): void {
     const is_plan_self_hosted = realm.realm_plan_type === 1;
     if (current_user.is_admin && !(is_plan_plus || is_plan_self_hosted)) {
         $("#realm_can_access_all_users_group_widget").prop("disabled", true);
-        const opts = {
+        let opts = {
             content: $t({
                 defaultMessage: "This feature is available on Zulip Cloud Plus. Upgrade to access.",
             }),
         };
+        if (!settings_data.user_has_billing_access()) {
+            opts = {
+                content: $t({
+                    defaultMessage: "This feature is available on Zulip Cloud Plus.",
+                }),
+            };
+        }
```

Comment:
`const opts = {}`, and then and if/else setting `opts.content = `, for a cleaner read.